apps/web/src/data-access/users.ts

```diff
@@ -1,20 +1,24 @@
 import {
+  getUser,
   Result,
   UserNotFoundError,
   users,
   verifyPassword,
-  type AppContext,
   type PromisedResult,
   type SessionData,
 } from '@latitude-data/core'
 import { getWorkspace } from '$/data-access/workspaces'
+import database from '$core/client'
```

Comment:
Importsss

Reply:
please by clear in comments don't know what you mean

Reply:
`@latitude-data/core`